Source/WebCore/css/CSSSelector.cpp

```diff
@@ -536,28 +536,20 @@ String CSSSelector::selectorText(StringView separator, StringView rightSide) con
             case PseudoElement::ViewTransitionImagePair:
             case PseudoElement::ViewTransitionOld:
             case PseudoElement::ViewTransitionNew: {
```

Comment:
No need for braces any more since we don’t have local variables. Same is true elsewhere above and below in this function.